cmd/buildah/pull.go

```diff
@@ -1,32 +1,31 @@
 package main
```

Comment:
Why restore these commands, This is not part of the API that OpenShift uses, is it?

Reply:
Functionally it's identical but letting the commands to continue using the API exercises them in CI.

Reply:
I think that's the right middle ground of preserving the API but moving the code over to libimage.